### PR TITLE
Feature/1.7/service remote description

### PR DESCRIPTION
**Pull request by lazan**

### CODE REVIEW

File: documentation/changelog.md [hunk 1]
```diff
@@ -1,6 +1,22 @@
...
 This is the changelog for `casual` and all changes are listed in this document.
 
+## [1.7.0]
+
+### Potentially breaking changes
+### CLI
+- `service --list-instances` column `state`, value `remote` has changed to `external` 
+   to be consistent with other CLI values. We use _internal_ and _external_, where 
+   _internal_ is stuff that is known locally to the "manager", and _external_ is stuff
+   is that is not known locally (could still be _external_ within a domain).
```
Comment (ahhud) [inline documentation/changelog.md]: Some extra _is_ here.

Comment (ahhud) [reply documentation/changelog.md]: And why are the builds failing? :O

Comment (lazan) [reply documentation/changelog.md]: I think it's because some sort of race-condition with requesting state. Hence, your expertise  :)

**Review by AlbsAgren — APPROVED** [review]
Looks good!

**Pull request merged**